Makefile

```diff
@@ -40,6 +40,8 @@ KLUSTERLET_KUBECONFIG_CONTEXT?=$(shell $(KUBECTL) config current-context)
 KLUSTERLET_KIND_KUBECONFIG?=$(HOME)/cluster1-kubeconfig
 HUB_KIND_KUBECONFIG?=$(HOME)/hub-kubeconfig
 KIND_CLUSTER?=kind
+MANAGED_CLUSTER?=cluster1
```

Comment:
What is the diff between MANAGED_CLUSTER and KIND_CLUSTER?

Reply:
They are actually the same. But I think it would be clearer to just use `MANAGED_CLUSTER` as `KIND_CLUSTER` is ambiguous (the hub cluster is also a kind cluster). If you are ok with that I will just use `MANAGED_CLUSTER` and get rid of `KIND_CLUSTER`.

Reply:
i think we can  use MANAGED_CLUSTER only